src/ServiceManagement/ExpressRoute/Commands.ExpressRoute/BgpPeerings/NewAzureBGPPeering.cs

```diff
@@ -36,6 +36,9 @@ public class NewAzureBGPPeeringCommand : ExpressRouteBaseCmdlet
             HelpMessage = "Peer Asn")]
         public UInt32 PeerAsn { get; set; }
 
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "Legacy Mode")]
```

Comment:
Will this parameter value really come from the pipeline?  What, exactly does LegacyMode mean?